src/api/config/routes.rb

```diff
@@ -174,7 +174,7 @@ def self.matches?(request)
       post 'project/create_image_repository' => :create_image_repository
 
       # Flags
-      post 'flag/:project(/:package)/:flag' => :toggle_flag, constraints: cons
```

Comment:
Seems like the URL helper produced wrong URLs and we ended up with

```
'flag/home:Admin/testpackage?flag=42
```
and then the controller interpreted ``testpackage`` as the flag id and tried to fetch ``Flag.find('testpackage')`` which obviously failed. In Project the URL was correct. But anyway, I think using `PUT` for `:update` and `POST` for `:create` and dropping the flag from the route is anyway more the rails way...